From 57d92ad35783ba523abea757be6543236e69a113 Mon Sep 17 00:00:00 2001
From: Matthias Leitl <a.dead.trousers@gmail.com>
Date: Sun, 1 May 2022 12:27:38 +0200
Subject: [PATCH 1/1] Have recovery as a possible target for vendor blobs

---
 extract_utils.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/extract_utils.sh b/extract_utils.sh
index a750f2c..2c0b5d2 100644
--- a/extract_utils.sh
+++ b/extract_utils.sh
@@ -322,6 +322,10 @@ function write_product_copy_files() {
             local OUTTARGET=$(truncate_file $TARGET)
             printf '    %s/proprietary/%s:$(TARGET_COPY_OUT_SYSTEM)/%s%s\n' \
                 "$OUTDIR" "$TARGET" "$OUTTARGET" "$LINEEND" >> "$PRODUCTMK"
+        elif prefix_match_file "recovery/" $TARGET ; then
+            local OUTTARGET=$(truncate_file $TARGET)
+            printf '    %s/proprietary/%s:$(TARGET_COPY_OUT_RECOVERY)/%s%s\n' \
+                "$OUTDIR" "$TARGET" "$OUTTARGET" "$LINEEND" >> "$PRODUCTMK"
         else
             printf '    %s/proprietary/%s:$(TARGET_COPY_OUT_SYSTEM)/%s%s\n' \
                 "$OUTDIR" "$TARGET" "$TARGET" "$LINEEND" >> "$PRODUCTMK"
-- 
2.25.1

